e2e/pages/app/toolbar-search/toolbar-search.testpage.component.ts

```diff
@@ -14,20 +14,37 @@ export class ToolbarSearchTestPageComponent {
     searchedFor: string = '';
     alwaysExpanded: boolean = false;
 
+    toolbarExpanded: boolean;
```

Comment:
I dont think the `searchText` variable above is used any more

Reply:
searchText is being used for the onSearch below and that's being used on toolbar search (search)="onSearch($event)

Reply:
The searchText in that function is being passed in, it is not using `this.searchText` which is the class property.

![image](https://user-images.githubusercontent.com/20795331/90140829-34332a00-dd72-11ea-978b-e0fab922477d.png)

Also the whole `onToolbarSearch` is not called from the HTML so it too can be removed?